prime-dotnet-webapi/Services/interfaces/IPlrProviderService.cs

```diff
@@ -0,0 +1,10 @@
+using System.Threading.Tasks;
+using Prime.Models;
+
+namespace Prime.Services
+{
+    public interface IPlrProviderService
+    {
+        public Task<int> CreateOrUpdatePlrProviderAsync(PlrProvider dataObject, bool expectExists = false);
```

Comment:
Just for the sake of being informative, access modifiers on C# interfaces have only been available since C# 8, In general, you don't need to specify access modifiers on Interface members because they are public